vpr/src/pack/cluster_legalizer.h

```diff
@@ -91,6 +92,11 @@ struct LegalizationCluster {
     ///        Contains information about the atoms in the cluster and how they
     ///        can be routed within.
     t_lb_router_data* router_data;
+
+    /// @brief The stats on where the different atoms in the cluster are currently
+    ///        placed in the cluster. This is what contains and decides which
```

Comment:
"this is what contains ..." -> "This is used when the legalizer decides what sites it should try to put a new molecule into."